lib/seed_migration/migrator.rb

```diff
@@ -228,9 +237,18 @@ def self.create_seed_file
             eos
           end
         end
-        file.write <<-eos
+        if !SeedMigration.disable_referential_integrity
+          file.write <<-eos
 end
-
+          eos
+        else
+          file.write <<-eos
+  end
+end
```

Comment:
we'll always need this `end`, so we could refactor the condition to make it a bit simpler:

```ruby
if SeedMigration.disable_referential_integrity
  file.write <<-eos
  end
  eos
end

file.write "end" # We don't need the heredoc here as this end doesn't require any indentation
```